src/ConnectionCard/ConnectionCardRow.js

```diff
@@ -0,0 +1,20 @@
+// @flow
+
+import * as React from 'react';
+import { View } from 'react-native';
+import StyleSheet from '../PlatformStyleSheet';
+import type { StylePropType } from '../PlatformStyleSheet/StyleTypes';
+
+type Props = {|
+  children: React.Node,
+  style?: StylePropType,
+|};
```

Comment:
read-only https://flow.org/en/docs/types/interfaces/#toc-interface-property-variance-read-only-and-write-only